### DISCUSSION TITLE
Sujets sur lesquelles on voudrait se faire sponsoriser

### DISCUSSION THREAD

**Discussion by annabreyer** [ladiesOfCodeParis/welcome-kit#6] [Ideas]
Lister les choses pour lesquelles on aimerait bien les sponsors

**Comment by annabreyer** [comment]
Frais de fonctionnement de LoC : 
Hébergement web + domaine 
Ça peut prendre deux formes : 
- argent 
- carrément l’hébergement (je pense notamment aux boites d’hébergement françaises comme ovh, plateforme.sh, scalingo ... )

**Comment by Morgane-Flauder** [comment]
- Upgrader le compte Slack pour avoir accès à tous les messages
- Goodies

**Comment by assuzzanne** [comment]
- Abonnement à un software de webinars
- Argent pour workshops ou apéros (pourquoi pas une soirée annuelle ou biannuelle LoC ?)

**Comment by pducolin** [comment]
- Payer des speakeuses expérimentées (remboursement déplacement, contribution talk) 
- Captions pour les talks en ligne
